packages/website/src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,36 @@
+import { Button } from '@coderscamp/ui/components/Button';
+import { Center } from '@coderscamp/ui/components/Center';
+import { Flex } from '@coderscamp/ui/components/Flex';
+import { HStack } from '@coderscamp/ui/components/Stack';
+import { useBreakpointValue } from '@coderscamp/ui/hooks/useBreakpointValue';
+
+import { DesktopBaseNavbar, MobileBaseNavbar } from './BaseNavbar';
+
+export const Navbar = () => {
+  const baseNavbar = useBreakpointValue({ base: <MobileBaseNavbar />, lg: <DesktopBaseNavbar /> } as const);
+  const buttonSize = useBreakpointValue({ base: 'xs', sm: 'md' } as const);
+  const horizontalPadding = useBreakpointValue({ base: '8px', sm: '18px', md: '24px', lg: '40px' });
```

Comment:
🙃 Padding supports responsive values so there is no need for `useBreakpointValue`